cmd/minikube/cmd/config/config.go

```diff
@@ -175,6 +175,12 @@ var settings = []Setting{
 		validations: []setFn{IsValidAddon},
 		callbacks:   []setFn{EnableOrDisableAddon},
 	},
+	{
```

Comment:
Making a change here will just expose another addon.. Not a config set. I could be wrong but can you double check.